### PR TITLE
Add some more UI clients to readme

### PR DESCRIPTION
**Pull request by fsproru**
First of all, thanks for making localstack happen. It's a great product, I use it on most of my projects.

It would greatly help the adoption of this project if developers have more visibility into each service. Managing the data in localstack is super useful for development.

Therefore, we build [Commandeer](https://getcommandeer.com) which works with localstack out of the box. Scanning for localstack is also a part of our onboarding.

This PR is just to start a conversation about a potential collaboration between Commandeer and Locastack. Let me know if you see more opportunities to collaborate.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -491,6 +491,11 @@ components and the relationship between them.
 localstack web
 ```
 
+## Other UI Clients
+
+* [Commandeer desktop app](https://getcommandeer.com)
+* (DynamoDB Admin Web UI)(https://www.npmjs.com/package/dynamodb-admin)
```
Comment (whummer) [inline README.md]: nit: Can we please fix the markup for the second link

**Comment by whummer** [comment]
Thanks for this PR @fsproru , happy to merge it. Would definitely be great to explore potential collaborations. I'll ping you directly on LinkedIn..

File: README.md (hunk 1 above)
Comment (fsproru) [reply README.md]: Good catch, updated.

**Comment by fsproru** [comment]
Thanks @whummer, I updated the markup for the second link. Let me know if you need anything else to proceed.

On a related note, I sent you a connection request on LinkedIn, looking forward to talking further.

**Pull request merged**